add docs for fixed packages (#764)
changesets/changesets: docs/dictionary.md

- **multi-package repo/monorepo** - A repository which contains multiple packages, generally managed by [Bolt](https://github.com/boltpkg/bolt) or [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/).
- **release line generators** - The `getReleaseLine` and `getDependencyReleaseLine` functions which are responsible for creating the lines inserted into changelog. A changelog entry for a particular release can be thought of as `releaseLineGenerators(changesets)`
- **linked packages** - Linked packages share a semver categorisation, such that all published linked packages have consistent new semver ranges. The logistics of this are best left to our [./linked-packages.md] documentation.

> - **fixed packages** - Fixed packages share a semver categorisation, such that all linked packages have consistent new semver ranges and will be published together. The logistics of this are best left to our [./fixed-packages.md] documentation.

Reviewer: ```suggestion
- **fixed packages** - Fixed packages share a semver categorisation, such that all fixed packages have the same semver version and are always published together. The logistics of this are best left to our [./fixed-packages.md] documentation.
```
